src/SignalR/clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -41,10 +41,8 @@
     "dist/**/*",
     "src/**/*"
   ],
-  "peerDependencies": {
-    "@aspnet/signalr": "^1.0.0-preview3"
-  },
   "dependencies": {
+    "@aspnet/signalr": "file:../signalr",
```

Comment:
@anurse since you were investigating this.

Reply:
We can't put a `file:` reference in here as-is. NPM doesn't translate this at all during packing (it ends up in the final package as a `file:` link to a path that doesn't exist). If we do this, we have to rewrite the dependency manually at pack time (in `eng/targets/Npm.targets`)

I couldn't find anything built-in to help us out here :(.

Reply:
There are tools like [Lerna](https://github.com/lerna/lerna) that some repos use to handle this, but I'm hesitant to bring a whole new tool into play here for this...

Reply:
I updated the Npm.targets to support rewriting the file. Unfortunately `ReadAllLines` returns a sequence of trimmed lines, so the formatting for the package.json is lost. Is that something to be concerned about?

Reply:
I am not worried about that. It's not ideal, but it's OK.